### DISCUSSION TITLE
Userprefs

### DISCUSSION THREAD

**Discussion by rubiin** [prasanthrangan/hyprdots#1845] [Q&A] [answered]
Suppose I want to override a keybind on keybindigs file.
Would adding the keybind on `userprefs` would work?
for example
```sh
// keybindigs.conf

bind = $mainMod, T, exec, $term # launch terminal emulator

```

```sh
// userprefs

bind = $mainMod, T, exec, ktorrent # launch ktorrent
```

**Comment by kRHYME7** [comment]
unbind it first 

https://wiki.hyprland.org/Configuring/Binds/#unbind

**Reply by rubiin** [reply]
Just found out. To avoid any potential issues , I am against editing the files `hyprdot` files so I want all my changes to go on userperfs

**Reply by rubiin** [reply]
could you update the wiki too. The repo wiki

**Reply by rubiin** [reply]
https://github.com/prasanthrangan/hyprdots/wiki/FAQ's-&-Issues#frequently-asked-questions this

**Comment by rubiin** [comment, marked as answer]
Turns out first I have to unbind or both these keybinds will execute
https://wiki.hyprland.org/Configuring/Binds/#unbind

**Reply by kRHYME7** [reply]
I made a script to generate and unbind duplicate keys a year a go, lol. That one is not accepted here, though.

**Reply by rubiin** [reply]
> I made a script to generate and unbind duplicate keys a year a go, lol. That one is not accepted here, though.

:D i thought it would override as well. because most things I use when a duplucate occurs does that

**Reply by kRHYME7** [reply]
I'm thinking of making a feature request for override-able key bind flag. Sounds dumb but QOL
